composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "ext-pcre": "*",
-        "php": ">=5.6.0"
+        "php": ">=7.2.0"
```

Comment:
"^7.2.0" rather for semver, php tries to follow that.

Reply:
```suggestion
        "php": "^7.2.0"
```

Reply:
Thanks for the suggestion, I fixed it.